src/script/components/InputBar/TypingIndicator/index.ts

```diff
@@ -17,6 +17,10 @@
  *
  */
 
+import {TIME_IN_MILLIS} from 'Util/TimeUtil';
+
 export * from './TypingIndicator';
 
 export {useTypingIndicatorState} from './TypingIndicator.state';
+
+export const IS_TYPING_TIMEOUT = TIME_IN_MILLIS.SECOND * 10;
```

Comment:
Why `IS`? I don't understand the naming here? 🤔

Reply:
agreed, fixed in https://github.com/wireapp/wire-webapp/pull/14165/commits/f0275432c211fda89aa4ea2af8a62da24db21e34